runtime/cfdumper/stubs.c

```diff
@@ -57,7 +57,11 @@ void freeMemorySegment (J9JavaVM *javaVM, J9MemorySegment *segment, BOOLEAN free
 J9Class *   
 internalCreateRAMClassFromROMClass(J9VMThread *vmThread, J9ClassLoader *classLoader, J9ROMClass *romClass,
 	UDATA options, J9Class* elementClass, j9object_t protectionDomain, J9ROMMethod ** methodRemapArray,
-	IDATA entryIndex, I_32 locationType, J9Class *classBeingRedefined, J9Class *hostClass)
+	IDATA entryIndex, I_32 locationType, J9Class *classBeingRedefined, J9Class *hostClass
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+	, J9Class *companionClass
```

Comment:
same here

Reply:
Won't compile in non-Valhalla build if comma moved on hostClass,